src/main/java/gov/ca/emsa/service/controller/PatientDiscoveryController.java

```diff
@@ -128,8 +133,29 @@ public String patientDiscovery(@RequestBody String request) throws InterruptedEx
 					patientPerson.getValue().getAdministrativeGenderCode().setCode("UN");
 				}
 				
-				LocalDate dob = LocalDate.parse(search.getDob(), DateTimeFormatter.ISO_DATE_TIME);
-				patientPerson.getValue().getBirthTime().setValue(dob.format(DateTimeFormatter.BASIC_ISO_DATE));
+				patientPerson.getValue().getBirthTime().setValue(search.getDob());
+				if(!StringUtils.isEmpty(search.getTelephone())){
+					TELExplicit tel = new TELExplicit();
+					tel.setValue("tel:+1-" + search.getTelephone());
```

Comment:
Agree with this comment. A simple check could be done to see if the telephone number in "search" already starts with tel:+1... but since it's mock it doesn't matter too much. Will approve either way.